core/vm/evm.go

```diff
@@ -602,6 +604,86 @@ func (evm *EVM) CallFromSystem(contractAddress common.Address, abi abipkg.ABI, f
 	return evm.handleABICall(abi, funcName, args, returnObj, call)
 }
 
+type cacheKey struct {
+	StateRoot       common.Hash
+	ContractAddress common.Address
+	TransactionData string
+}
+
+type cacheResult struct {
+	ret     []byte
+	gasLeft uint64
+}
+
+var staticCallCache, _ = lru.New(100)
+var (
+	cacheHits    = metrics.NewRegisteredMeter("contract_comm/caller/cache_hits", nil)
```

Comment:
I would suggest a namespace relevant to the `core/vm` package, but there aren't currently any others. I would suggest `contract_comm/systemcall` just to align with the ones in `contract_comm/evm` though
```suggestion
	cacheHits    = metrics.NewRegisteredMeter("contract_comm/systemcall/cache_hits", nil)
```